apps/dash-web-trader/app.py

```diff
@@ -0,0 +1,1447 @@
+                            dcc.Checklist(
+                                id=pair + "studies",
+                                options=[
+                                    {
+                                        "label": "Accumulation/D",
+                                        "value": "accumulation_trace",
+                                    },
+                                    {
+                                        "label": "Bollinger bands",
+                                        "value": "bollinger_trace",
+                                    },
+                                    {"label": "MA", "value": "moving_average_trace"},
+                                    {"label": "EMA", "value": "e_moving_average_trace"},
+                                    {"label": "CCI", "value": "cci_trace"},
+                                    {"label": "ROC", "value": "roc_trace"},
+                                    {"label": "Pivot points", "value": "pp_trace"},
+                                    {
+                                        "label": "Stochastic oscillator",
+                                        "value": "stoc_trace",
+                                    },
+                                    {
+                                        "label": "Momentum indicator",
+                                        "value": "mom_trace",
+                                    },
+                                ],
+                                values=[],
+                            )
+                        ],
+                        style={"display": "none"},
+                    ),
+                    # Styles checklist
+                    html.Div(
+                        id=pair + "style_tab",
+                        children=[
+                            dcc.RadioItems(
+                                id=pair + "chart_type",
+                                options=[
+                                    {
+                                        "label": "candlestick",
+                                        "value": "candlestick_trace",
+                                    },
+                                    {"label": "line", "value": "line_trace"},
+                                    {"label": "mountain", "value": "area_trace"},
+                                    {"label": "bar", "value": "bar_trace"},
+                                    {
+                                        "label": "colored bar",
+                                        "value": "colored_bar_trace",
+                                    },
+                                ],
+                                value="colored_bar_trace",
+                            )
+                        ],
+                    ),
+                ],
+            ),
+            # Chart Top Bar
+            html.Div(
+                className="row chart-top-bar",
+                children=[
+                    html.Span(
```

Comment:
A dropdown would be a more intuitive in this case. To achieve the same look, you can change the dropdown style.